autogpt_platform/backend/schema.prisma

```diff
@@ -635,9 +639,7 @@ model StoreListingVersion {
   agentVersion Int
   Agent        AgentGraph @relation(fields: [agentId, agentVersion], references: [id, version])
 
-  // The details for this version of the agent, this allows the author to update the details of the agent,
-  // But still allow using old versions of the agent with there original details.
-  // TODO: Create a database view that shows only the latest version of each store listing.
+  // Content fields
   slug        String
```

Comment:
Should we make these optional ?

Reply:
why should they be? slug maybe should be on the main storelisting?